pkgs/applications/audio/netease-cloud-music-gtk/update-cargo-lock.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p coreutils ripgrep git cargo
+
+# Ref: https://github.com/NixOS/nixpkgs/blob/nixos-21.05/pkgs/applications/audio/netease-music-tui/update-cargo-lock.sh
+
+set -eu -vx
+
+here=$PWD
+version=$(cat default.nix | rg '^  version = "' | cut -d '"' -f 2)
```

Comment:
Please use normal grep. Also grep/rg can directly read a file, no need to cat pipe it.